bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/api/dto/Result.java

```diff
@@ -12,13 +12,16 @@
  */
 package org.openhab.binding.evcc.internal.api.dto;
 
+import java.util.Map;
+
 import com.google.gson.annotations.SerializedName;
 
 /**
- * This class represents the result object of the status response (/api/state).
- * This DTO was written for evcc version 0.117.0
+ * This class represents a loadpoint object of the status response (/api/state).
```

Comment:
I think it still represents the "main body" of the response, i.e. the result object.

Reply:
ack